pkg/datapath/linux/config/config.go

```diff
@@ -543,6 +547,9 @@ func (h *HeaderfileWriter) writeStaticData(fw io.Writer, e datapath.EndpointConf
 		callsMapName = callsmap.HostMapName
 	}
 	fmt.Fprintf(fw, "#define CALLS_MAP %s\n", bpf.LocalMapName(callsMapName, epID))
+	if option.Config.EnableCustomCalls && !e.IsHost() {
```

Comment:
I think we also need to skip some logic in case of host endpoint during the object file patching. Otherwise we'll get warnings. I believe the CI tests should catch this.

Reply:
Specifically I'd expect this:

```
scopedLog.Warning("Skipping symbol substitution")
```